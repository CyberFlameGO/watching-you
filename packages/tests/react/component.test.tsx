import React from 'react';
import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import WatchingYou from '../../react';//'react-watching-you';
import { mousemove } from '../utils';

describe('Components', () => {
  it('It should work', async () => {
    render(
      // @ts-ignore
      <WatchingYou>
        <div>O</div>
      </WatchingYou>,
    );
    await mousemove(100, 100);
    expect(document.body).toMatchSnapshot();
    await mousemove(400, 200);
    expect(document.body).toMatchSnapshot();
  });
});
