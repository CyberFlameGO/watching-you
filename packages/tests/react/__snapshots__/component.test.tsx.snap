// Vitest Snapshot v1

exports[`Components > It should work 1`] = `
<body>
  <div>
    <div
      style="transform: translate3d(39.7px,30.4px,0px) rotate(127.5deg);"
    >
      O
    </div>
  </div>
</body>
`;

exports[`Components > It should work 2`] = `
<body>
  <div>
    <div
      style="transform: translate3d(45.2px,21.4px,0px) rotate(115.4deg);"
    >
      O
    </div>
  </div>
</body>
`;
